meta-facebook/yv4-sd/src/platform/plat_pldm_monitor.h

```diff
@@ -17,7 +17,11 @@
 #ifndef PLAT_PLDM_MONITOR_H
 #define PLAT_PLDM_MONITOR_H
 
-#define MAX_STATE_EFFECTER_IDX 168
+#define MAX_STATE_EFFECTER_IDX 169
+
+#define PLDM_PLATFORM_HOST_PWR_CTRL_DEFAULT 0xFF
```

Comment:
Lets use consistent prefix in code, since other places its plat_pldm, so let use PLAT_PLDM_

Reply:
Revised, thank you for correction.